test/DevicesController.test.js

```diff
@@ -41,20 +41,22 @@ test.before(async () => {
     });
 
   userToken = tokenResponse.body.access_token;
-});
 
-// TODO come up better test name
-test.serial('provision test', async t => {
-  const response = await request(app)
+  if (!userToken) {
+    throw new Error('test user creation fails');
```

Comment:
if testUser creation fails, this prevent tests execution in the file. it helps to get clean errors when something wrong with user creation/auth flow.